presets/ragengine/main.py

```diff
@@ -278,6 +282,81 @@ async def query_index(request: QueryRequest):
         rag_query_requests_total.labels(status=status).inc()
         rag_query_latency.labels(status=status).observe(time.perf_counter() - start_time)
 
+@app.post(
+    "/v1/chat/completions",
+    response_model=ChatCompletionResponse,
+    summary="OpenAI-Compatible Chat Completions API",
+    description="""
+    OpenAI-compatible chat completions endpoint with RAG capabilities.
+
+    ## Request Example:
+    ```json
+    {
+      "model": "example_index",
```

Comment:
In this api document, do you need to add index_name here? This is the key for user to use this api.